.github/workflows/release_changelog.yml

```diff
@@ -1,4 +1,4 @@
-name: 'Releases'
+name: 'Release changelog'
```

Comment:
Just a rename. Keeping this workflow relying on tags because it uses a deprecated action and relies on tag refs. We should migrate this: https://github.com/Unleash/unleash/actions/runs/9125610662/job/25092161594#step:4:3